File: MyProjects/MAP/Assignment 2/src/View/Command/RunExample.java
package View.Command;

import Controller.Controller;

public class RunExample extends Command{
    private Controller contro;
    public RunExample(String key,String descr,Controller c){
        super(key,descr);
        this.contro=c;
    }

    @Override
    public void execute(){
        try{
            contro.allSteps();
        }
        catch(Exception e){
            System.out.println(e);
        }
    }
}

File: MyProjects/MAP/Assignment 7/Interpreter/View/RunExample.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Interpreter.View
{
    public class RunExample : Command
    {
        private Controller.MyController ctrl;
        public RunExample(string key, string desc, Controller.MyController ctr) : base(key, desc)
        {
            this.ctrl = ctr;
        }

        public override void execute()
        {

            try
            {
                ctrl.executeAllSteps();
            }
            catch (SystemException e)
            {
                Console.WriteLine(e.Message);
            }
        }

    }
}
